src/videojs.ima.js

```diff
@@ -541,7 +541,7 @@
         function( contentSrc, adTag, playOnLoad) {
       player.ima.resetIMA_();
       settings.adTagUrl = adTag ? adTag : settings.adTagUrl;
-      player.pause();
+      player.tech && player.pause();
```

Comment:
What is player.tech? It's not listed in [official video.js API docs](https://github.com/videojs/video.js/blob/stable/docs/api/vjs.Player.md).